it/test/uk/gov/hmrc/excisemovementcontrolsystemapi/repository/UnusedRepositoryRemoverStartItSpec.scala

```diff
@@ -57,25 +58,32 @@ class UnusedRepositoryRemoverStartItSpec
   }
 
   private def createIfCollectionDoesNotExist(collectionName: String, collectionExists: Boolean) =
-    if (!collectionExists) mongoComponent.database.createCollection(collectionName).toFuture()
+    if (!collectionExists) {
+      println(s"*** creating collection name $collectionName")
+      mongoComponent.database.createCollection(collectionName).toFuture()
+    }
     else Future.successful()
 
   "unusedRepositoryRemover" should {
-    "drop repository miscoded movements archive on class instantiation" in {
-      val collectionName = "miscoded-movements-archive"
+    "drop repository problem movements archive on class instantiation" in {
+      val collectionName = "movements-archive"
       await(existsCollection(mongoComponent, collectionName)) mustBe false
     }
 
-    "drop repository miscoded movements workItems on class instantiation" in {
-      val collectionName = "miscoded-movements-workItems"
+    "drop repository problem movements workItems on class instantiation" in {
+      val collectionName = "problem-movements-workItems"
       await(existsCollection(mongoComponent, collectionName)) mustBe false
     }
   }
+
   def existsCollection(
-    mongoComponent: MongoComponent,
-    collectionName: String
-  ): Future[Boolean] =
+                        mongoComponent: MongoComponent,
+                        collectionName: String
+                      ): Future[Boolean] =
     for {
       collections <- mongoComponent.database.listCollectionNames().toFuture()
-    } yield collections.contains(collectionName)
+    } yield {
+      println(s"*** checking for $collectionName collections present: $collections")
```

Comment:
println

Reply:
Yes these are here on purpose, I have had to return it to the state it was in when the tests were passing :/ this test is not cooperating at all :(